test/id.test.js

```diff
@@ -247,3 +247,182 @@ describe('mongodb default id name', function() {
     );
   });
 });
+
+describe('strictObjectIDCoercion', function() {
```

Comment:
We already have tests for strictObjectIDCoercion in [test/objectid.test.js](https://github.com/strongloop/loopback-connector-mongodb/blob/master/test/objectid.test.js).

Have you considered placing your new test to that file instead? If you prefer to keep them in `test/id.test.js`, then what rule should we use to decide where to put new ObjectID-related tests in the future (`test/id.test.js` vs. `test/objectid.test.js`)?

Equally important, please check the existing tests in `test/objectid.test.js` file and make sure your new tests are not duplicating any existing tests.

Reply:
Initially started with `test/objectid.test.js`, but moved to `test/id.test.js` because the tests were id-specific rather than ObjectID generic. May move it back eventually, let me focus on the implementation for now.